pkg/client/portforward/portforward_test.go

```diff
@@ -206,6 +206,88 @@ func (s *fakeUpgradeStream) Headers() http.Header {
 	return http.Header{}
 }
 
+func CloseListener(l net.Listener) {
```

Comment:
oh, yes, sorry
Added it to properly handle closing in Code Review 5, forgot to remove it.
Will amend my last commit